automod/rules/replies.go

```diff
@@ -78,6 +78,39 @@ func IdenticalReplyPostRule(c *automod.RecordContext, post *appbsky.FeedPost) er
 	return nil
 }
 
+// Similar to above rule but only counts replies to the same post. More aggressively applies a spam label to new accounts that are less than a day old.
+var identicalReplySameParentLimit = 3
+var identicalReplySameParentSpamLabel = 24 * time.Hour
+var _ automod.PostRuleFunc = IdenticalReplyPostSameParentRule
+
+func IdenticalReplyPostSameParentRule(c *automod.RecordContext, post *appbsky.FeedPost) error {
+	if post.Reply == nil || IsSelfThread(c, post) {
+		return nil
+	}
+
+	if ParentOrRootIsFollower(c, post) {
+		return nil
+	}
+
+	period := countstore.PeriodDay
+	bucket := c.Account.Identity.DID.String() + "/" + post.Reply.Parent.Uri + "/" + HashOfString(post.Text)
+	c.IncrementPeriod("reply-text-same-post", bucket, period)
```

Comment:
nice, this is a bit subtle but you did the right thing: the cardinality of (poster, URI, hash) is going to be huge, but by using `IncrementPeriod` (instead of generic increment), we limit to only last-hour bucket and the counters won't stick around.

Reply:
tweaked this to hour, think that makes sense.